readme.md

```diff
@@ -19,7 +19,7 @@ To prevent arbitrary code execution attacks, AutoSource will prompt you to appro
 This plugin does not yet have tests and so this feature can not be guaranteed.
 
 ## Prerequisistes
-AutoSource depends `shasum` being installed on your system. MacOS must also have `uuid` and Linux must have `uuidgen`.
+AutoSource depends `shasum` being installed on your system.
```

Comment:
Dope. I'll be swapping out the `shasum` call to the `shasum()` builtin later today so this plugin should have no deps later today 🙌